### PR TITLE
Option flags for SSH key path, SSH key name and password file name

### PR DESCRIPTION
**Pull request by x3dfxjunkie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 ._*
```
Comment (ahnick) [inline .gitignore]: I don't think these file extensions need to be ignored as the code repository should not be cloned into where a user's script resides.  Rather the encpass.sh script right now should be copied out of the folder of the cloned repo and into the folder where the script resides.  Eventually the installation instructions will be changed to have the user add encpass.sh to the user's path as encpass.sh will support holding the keys in a directory (.encpass) under the user's home directory.  Also, on the off chance I need to include a file with these extensions I'd rather not add them to the .gitignore.

File: README.md [hunk 2]
```diff
@@ -31,12 +31,19 @@ See the test.sh example...
 . ./encpass.sh
 password=$(get_password)
 # Call it specifying a directory
-#password=$(get_password ~/.ssh)
+#password=$(get_password -f ~/.ssh)
 echo $password
 ```
 
-## Limitations
```
Comment (ahnick) [inline README.md]: We do not want to get rid of POSIX compliance and for reasons outlined elsewhere we don't want to introduce options for SSH right now as the SSH dependency will be going away.

File: encpass.sh [hunk 3]
```diff
@@ -21,76 +21,127 @@
...
+  if [[ "${PASSWORD}" = "${CPASSWORD}" ]]; then
+    echo "${PASSWORD}" | openssl rsautl -encrypt -pubin -inkey \
+      "${KEY_NAME}".pub.pem -out "${PASS_FILE}".enc
+  else
+    echo "Error: passwords do not match.  Please try again." >&2
+    exit 1
+  fi
 }
 
 get_abs_filename() {
```
Comment (ahnick) [inline encpass.sh]: why are you changing the get_abs_filename function?

File: encpass.sh [hunk 4]
```diff
@@ -21,76 +21,127 @@
...
-	# Create a PKCS8 version of the public key in the current directory if one does not already exist
-	if [ ! -e id_rsa.pub.pem ]; then
-		if [ ! -x "$(command -v ssh-keygen)" ]; then
-			echo "ssh-keygen is needed to generate a PKCS8 version of your public key.  Please install it and try again." >&2
-		fi
+  # Create a PKCS8 version of the public key in the current directory if one
+  # does not already exist
+  if [[ ! -f "${KEY_NAME}.pub.pem" ]]; then
+    if [[ ! -x "$(command -v ssh-keygen)" ]]; then
+      /bin/echo -n "ssh-keygen is needed to generate a PKCS8 version of" >&2
```
Comment (bebehei) [inline encpass.sh]: Could you please enlighten me, why you use `/bin/echo` instead of `echo`?

**Pull request closed without merging**